### ISSUE TITLE
meta: review old bg1 reports

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#563]
Recheck and report as individual issues if present.

>   * [landri.dlg; data/compiler issue](https://www.gibberlings3.net/forums/topic/21418-bugs-in-bg1-tosc-gemrb-063/?do=findComment&comment=179712)
>   * RESEARCH Wrong sound effect played if no arrows left- same as no charge in wand. Which one should it be?
>   * You can't travel from the Gnoll Stronghold to Wyrm's Crossing) RECHECK RESEARCH  (also, was it possible in the original?)
>   * Misaligned (?) Potion of Heroism (POTN09) use animation.
>   * //This is especially strange, it happens only in Taerom's shop (blackened scroll counter) [[http://img197.imageshack.us/img197/5888/201212311520071440x900s.png|009]]// RECHECK / STILL Similar thing in Kelddath shop at temple.
>   * ...lycanthropy infection works correctly (however I can replicate it only with one save, strange), turning other party members to greater wolfweres (greaterwolf), however they probably shouldn't drop Peladan as loot ;) 
>   * RECHECK dead party members shouldn't be visible after area transitions (!IF_ACTIVE or avatar removal). Speaking of travelling corpses, that's the case when Shandalar teleports party. I've checked in original, there are no visible corpses of (already dead) party members on island upon arrival, not the case with GemRB.
>   * Being magically asleep not using proper stance (lying, at least male human fighter) for the selected actor - eg. stinking cloud induced. Switches properly if you try to do anything, but should happen before.
>  
> Problematic BG1 saves: https://www.dropbox.com/sh/vp1h2atm08w42v8/C5T6GROoN3 (for some of the above)

**Comment by caillean7** [comment]
> - landri.dlg; data/compiler issue

 Works.

> - RESEARCH Wrong sound effect played if no arrows left- same as no charge in wand. Which one should it be?

The original doesn't play a sound at all.

> You can't travel from the Gnoll Stronghold to Wyrm's Crossing) RECHECK RESEARCH (also, was it possible in the original?)

It's possible on both engines; so all good here.

> - Misaligned (?) Potion of Heroism (POTN09) use animation.

See #582

**Comment by lynxlynxlynx** [comment]
scrolls: actual scrolls (scroll icons) in the shop interface. It was an odd thing, irregular.

**Comment by caillean7** [comment]
Concerning problematic save 1: The issue here is that in AR5900 (random encounter area with lots of different spawns depending on daytime) all the spawned creatures are instantly killed. Interestingly there are all enemies present from all the different hours. However, I could only reproduce that with this save - using another one works fine, no instant deaths in that area and the spawns are according to the time of day. So this is perhaps just a messed up save game?

**Comment by caillean7** [comment]
@lynxlynxlynx: I'll go back to the thing with lycantrophy tomorrow.

Also, noticed two other things: ghouls use wrong animation (send you a PR to fix), and there's something off with travel times (will do some more tests and open a new issue tomorrow / in the next couple of days).

**Comment by lynxlynxlynx** [comment]
Great, thanks!

**Comment by caillean7** [comment]
I checked out all points from the list, so unless you want to check out the thing about the problematic saves some more, it can probably be closed.

**Comment by lynxlynxlynx** [comment]
Thanks, closing! :fireworks: 
If you find something tangible about the travel times thing, do open a new report.
